setup.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+import re, io
+
+__version__ = re.search(
+    r'__version__\s*=\s*[rRfFuU]{0,2}[\'"]([^\'"]*)[\'"]',
+    io.open('msal_extensions/__init__.py', encoding='utf_8_sig').read()
+    ).group(1)
+
+setup(
+    name='msal-extensions',
+    version=__version__,
+    packages=find_packages(),
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+    ],
+    extra_require={
+        'dev': [
+            'pytest',
```

Comment:
Mind confirming whether or not `pytest` is your preferred way to author/execute tests?

Reply:
I believe there is less a need to standardize the test framework, so I wasn't even paying much attention to this line. :-P You can choose [whatever](https://docs.python-guide.org/writing/tests/) you want.

But then you kind of remind me, this `pytest` line would better NOT be put inside `extraS_require` (you had a typo there, by the way). [The `extras_require` are mean to organize extra FUNCTIONALITY, explained here](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies).

For test dependency, they can be put inside a different parameter, [`tests_require`](https://setuptools.readthedocs.io/en/latest/setuptools.html), so it would look like this: `tests_require=['pytest']`. For example, this is [what PyJWT did](https://github.com/jpadilla/pyjwt/blob/master/setup.py#L72).

Reply:
Another derpy moment! Fixed in 081933c